common/config/config.rb

```diff
@@ -15,4 +15,9 @@
 
 # Enable / disable PUI resource/archival object page actions
 AppConfig[:pui_page_actions_cite] = false
-AppConfig[:pui_page_actions_request] = false
\ No newline at end of file
+AppConfig[:pui_page_actions_request] = false
+AppConfig[:db_url] = "jdbc:mysql://localhost:3306/archivesspace?user=root&password=root@123&useUnicode=true&characterEncoding=UTF-8&useSSL=false"
```

Comment:
Remove `AppConfig[:db_url]` from config.rb for now. It will be loaded in by environment variables later on.